.circleci/config.yml

```diff
@@ -12,13 +12,15 @@ jobs:
 
   checkESLint:
     docker:
-      - image: node:slim
+      - image: node:18
     resource_class: large      
     steps:
       - checkout
       - run:
+          environment:
+            YARN_ENABLE_IMMUTABLE_INSTALLS: false
```

Comment:
Sometimes was getting some weird things with yarn giving errors for 'you cant change the lockfile', so needed to add this